src/js/controls.js

```diff
@@ -0,0 +1,32 @@
+const Clappr = require('clappr')
+const MediaControl = require('../html/NewMediaControl.html')
+const styles = require('styles/controls.less')
+
+class ParatiiMediaControl extends Clappr.MediaControl {
+  get template() {
+    return Clappr.template(
+      MediaControl
+    )
+  }
+  constructor(options = {}) {
+    super(options);
+  }
+
+  initializeIcons() {
+    super.initializeIcons();
+    this.$playPauseToggle.removeClass('video-paused');
```

Comment:
This is something that I think should be taken care of by `Clappr`. They place a `.paused` class on the player button but do not remove/re-add it correctly. I think I will open a PR to `Clappr` to fix this, but for now we can use this workaround.